RELEASE/scripts/sl_ascend.ash

```diff
@@ -3920,6 +3920,7 @@ boolean L11_palindome()
 			{
 				use(1, $item[&quot;2 Love Me\, Vol. 2&quot;]);
 				doHottub();
+				print("Torporing, since it looks like we're dead from reading a book.", "blue");
```

Comment:
Shouldn't this be conditional? Seems like it would print out of path too.

Reply:
smart